src/main/webapp/script.js

```diff
@@ -198,16 +198,32 @@ function convertRangeToString(range) {
 
 /**
  * Display loading image
+ * @param {string} idPrefix id of div to be altered
```

Comment:
Change this to: idPrefix prefix of div id that needs to be altered, idPrefix only has the prefix of the id.